### PR TITLE
Use a separate folder for every network under ~/.kaspawallet

### LINKED ISSUE #1669: Kaspawallet should keep a separate keys.json file per network
Currently, no matter how one runs kaspawallet, the default keys file location is `~/.kaspawallet/keys.json`.
If a user wants to keep separate wallets for e.g. testnet and devnet, he would have to type a non-default path for one of the key files, otherwise the file would be overriden.

This is pretty dangerous, and could easily lead to loss of coins.

I suggest keeping a separate file per network, using the same scheme kaspad uses (e.g. `~/.kaspawallet/kaspa-devnet/keys.json`, `~/.kaspawallet/kaspa-testnet/keys.json`, etc.).

### PR DESCRIPTION
**Pull request by svarogg**
This fixes #1669

### CODE REVIEW

File: cmd/kaspawallet/keys/keys.go [hunk 1]
```diff
@@ -157,7 +162,7 @@ func createFileDirectoryIfDoesntExist(path string) error {
 		return nil
 	}
 
-	return os.MkdirAll(dir, 0600)
+	return os.MkdirAll(dir, 0700)
```
Comment (elichai) [inline cmd/kaspawallet/keys/keys.go:165]: Why this change?

**Review by someone235 — APPROVED** [review]

File: cmd/kaspawallet/keys/keys.go (hunk 1 above)
Comment (elichai) [reply cmd/kaspawallet/keys/keys.go:165]: Manually checked and the permissions end up correct:
1. directories need "execution" permissions (to enter).
2. files are still 0600.

**Review by elichai — APPROVED** [review]
